packages/eslint-plugin/src/rules/prefer-find.ts

```diff
@@ -0,0 +1,110 @@
+import type { TSESLint } from '@typescript-eslint/utils';
+import { AST_NODE_TYPES } from '@typescript-eslint/utils';
+import * as tsutils from 'ts-api-utils';
+
+import { createRule, getParserServices } from '../util';
+
+export default createRule({
+  name: 'prefer-find',
+  meta: {
+    docs: {
+      description:
+        'Enforce the use of Array.prototype.find() over Array.prototype.filter() followed by [0] when looking for a single result',
+      requiresTypeChecking: true,
+      recommended: 'stylistic',
```

Comment:
[Bug] Yeah this would be a breaking change 😞 so we can't just yet. We don't have a "stylistic-strict" ruleset so there's really nowhere for this to go. 

The good news is, we're starting to ramp up on work for v7, so this won't be present-but-not-in-any-config for long!

Reply:
Removed